app/views/new_threads/list.html.erb

```diff
@@ -22,6 +22,10 @@
               <p><%= truncate_html(new_thread.description.html_safe) %> (<%= link_to "Read more", new_thread, :class => "thread-link"%>)</p>
             </div>
 
+            <div class="thread-foot">Tags : 
```

Comment:
Add the selectize css classes here to design the tags similar to that in the autocomplete.